[2025-03-31T02:04:50.963+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: covid_pipeline.download_data manual__2025-03-31T02:04:48.681304+00:00 [queued]>
[2025-03-31T02:04:50.966+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: covid_pipeline.download_data manual__2025-03-31T02:04:48.681304+00:00 [queued]>
[2025-03-31T02:04:50.966+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2025-03-31T02:04:50.974+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): download_data> on 2025-03-31 02:04:48.681304+00:00
[2025-03-31T02:04:50.978+0000] {standard_task_runner.py:57} INFO - Started process 2551 to run task
[2025-03-31T02:04:50.981+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'covid_pipeline', 'download_data', 'manual__2025-03-31T02:04:48.681304+00:00', '--job-id', '16', '--raw', '--subdir', 'DAGS_FOLDER/covid_pipeline.py', '--cfg-path', '/tmp/tmpferducb0']
[2025-03-31T02:04:50.982+0000] {standard_task_runner.py:85} INFO - Job 16: Subtask download_data
[2025-03-31T02:04:51.001+0000] {task_command.py:410} INFO - Running <TaskInstance: covid_pipeline.download_data manual__2025-03-31T02:04:48.681304+00:00 [running]> on host 246b607dca19
[2025-03-31T02:04:51.120+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='covid_pipeline' AIRFLOW_CTX_TASK_ID='download_data' AIRFLOW_CTX_EXECUTION_DATE='2025-03-31T02:04:48.681304+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-31T02:04:48.681304+00:00'
[2025-03-31T02:04:51.554+0000] {logging_mixin.py:150} INFO - Data downloaded!
[2025-03-31T02:04:51.555+0000] {python.py:183} INFO - Done. Returned value was: None
[2025-03-31T02:04:51.558+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=covid_pipeline, task_id=download_data, execution_date=20250331T020448, start_date=20250331T020450, end_date=20250331T020451
[2025-03-31T02:04:51.584+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2025-03-31T02:04:51.594+0000] {taskinstance.py:2653} INFO - 1 downstream tasks scheduled from follow-on schedule check
